inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl

```diff
@@ -258,7 +258,8 @@
         </ul>
       </div> -->
         <a class="btn btn-default pull-right" href="/search?p=refersto:{{record['control_number']}}&cc=HEP">
-            View all {% if record.get('citation_count', 0) > 0  %} {{ record.get('citation_count', '') }} {% endif %} citations</a>
+            View all {% if record.get('citation_count', 0) > 0  %} {{ record.get('citation_count', '') }} {% endif %} citations
```

Comment:
Maybe this can become a filter? Also, when `citation_count` is 0 the link is not very useful, so why show it?